docker/templates/macros.m4

```diff
@@ -71,6 +71,7 @@ libicu66 m4_dnl
 libpq5 m4_dnl
 libssl1.1 m4_dnl
 libxml2 m4_dnl
+moreutils m4_dnl
```

Comment:
Thanks -- I didn't have this installed on my local system, so it helped me figure out what package I needed. :)

I saw you mention you didn't modify any scripts for mirror servers, so it sounds like this shouldn't create any additional dependency for people outside of production.

Reply:
Right, it is not required for mirrors, but it is required for development if one wants to test running scripts using `ts`, so I opened https://github.com/metabrainz/musicbrainz-docker/pull/221.